### PR TITLE
#216 - Fix content to have example

### PR DESCRIPTION
**Pull request by ivanjunckes**

### CODE REVIEW

**Comment by EricWittmann** [comment]
OK - a couple of things here.  First, in order to contribute to **eclipse/microprofile-open-api** you must use the git "Signed Off By" feature for your commits.  See here:

https://dev.eclipse.org/eclipse-webhook/services/status_details.php?id=5b47ffc235a1f

So unfortunately I think you'll need to re-do the commit and resubmit.

That might be OK though, because the spec says that "example" and "examples" are mutually exclusive properties.  So I'm wondering if the TCK test class `ReviewResource` should have a `@Content` annotation with **both** `example` and `examples` properties.  It's unclear to me whether the TCK should be requiring implementations process annotations in that way, or if implementations are free to process the annotations in a way that ensures a valid OpenAPI document result.

In other words, an implementation might want to ignore the `example` property in cases where an `examples` property is also provided.

@arthurdm thoughts?

I looked at the other tests for examples here:  https://github.com/eclipse/microprofile-open-api/blob/master/tck/src/main/java/org/eclipse/microprofile/openapi/tck/AirlinesAppTest.java#L650-L665

And I don't see any other tests at all for the `example` property.

**Comment by ivanjunckes** [comment]
Opened another one, please see if it is ok with the IP.

https://github.com/eclipse/microprofile-open-api/pull/239

**Comment by EricWittmann** [comment]
I think this one can be closed now - the new PR is great.
